src/components/AddBook.js

```diff
@@ -1,17 +1,52 @@
-import React from 'react';
-import './App.css';
+// import React from 'react';
+// import './App.css';
+
+// function AddBook() {
+//   return (
+//     <div className="add-new-book" onSubmit={(e) => submitBookToStore(e)}>
+//       <h2 className="title">ADD NEW BOOK</h2>
+//       <input type="text" />
+//       <select name="cars" id="cars">
+//         <option value="Under Construction">Categories</option>
+//       </select>
+//       <input type="submit" value="ADD BOOK" />
+//     </div>
+//   );
+// }
+
+// export default AddBook;
+
+// // End
```

Comment:
- [ ] Kindly remove commented code as they make your codebase look messy and hard to read

Reply:
I have removed them, thanks

Reply:
Done- thanks